docs/common/craft-parts/how-to/plugins/nil_plugin.rst

```diff
@@ -0,0 +1,28 @@
+.. _how_to_nil_plugin:
```

Comment:
So I thought about this and I decided that I don't like this structure of the plugin examples being in how-tos. The example given here is more of an example of using an unsupported build tool to build a part, which is related to the `nil` plugin but then the plugin itself isn't the focus of the example. How-tos are also about being a series of steps/instructions towards doing something, like the how-to guide we have for [overriding a part's build](https://canonical-craft-parts.readthedocs-hosted.com/en/latest/common/craft-parts/how-to/override_build.html) (this example would be a better fit there than here, imo).

Finally, Diátaxis [likes examples in references](https://diataxis.fr/reference/#provide-examples) and the example that it gives (this is getting very meta) is "illustrating a command through an example of its usage" is pretty much our use-case for the plugins docs.

I'm happy to discuss this more if you want

Reply:
I removed this how-to for now